webpack/assets/javascripts/react_app/redux/actions/statistics.js

```diff
@@ -0,0 +1,19 @@
+import API from '../../API';
```

Comment:
still using jquery for ajax calls, i figure maybe this should be a seperate issue and discussion.